Automattic/wp-calypso
# Fixes the stats presentation for mobile of the Advertising posts page

Related to SELFSERVE-623

## Proposed Changes

We are showing -1 likes in the stats for the posts on the Advertising page. It doesn't happen on desktop.

This PRs uses the same logic as desktop to present the stats for mobile, and also fixes an alignment issue we have with the Dates.

Before:
![Screenshot 2023-06-22 at 17 34 30](https://github.com/Automattic/wp-calypso/assets/1258162/a16637fe-d031-46dc-8c23-de62855dbb98)

After:
![Screenshot 2023-06-22 at 17 36 27](https://github.com/Automattic/wp-calypso/assets/1258162/3175e702-08ce-4d7d-869e-4eb6b78b3ad4)

## Testing Instructions

You will need to have a site with stats/likes disabled. An easy way to test this is by creating a new Jurrasic Ninja site and connect it with Jetpack.

* Open the Live preview and navigate to Tools->Advertising
* Shrink the screen to mobile, and check that the links isn't show as -1
* Check that the Post date is correctly alinged.

## Pre-merge Checklist
